apps/judicial-system/xrd-api/src/app/app.controller.ts

```diff
@@ -71,15 +71,16 @@ export class AppController {
     }
   }
 
-  @Patch('subpoena/:subpoenaId')
+  // update by police subpoena id
+  @Patch('subpoena/:policeSubpoenaId')
```

Comment:
We need to ensure to regression test this on dev due to the rename. This should not be a breaking change for external users? since they just concat the id in the url, right?